cmd/asset/email.default.html

```diff
@@ -48,7 +48,26 @@ <h1 style="font-size: 32px; font-weight: 800; line-height: 32px; margin: 48px 0;
 
         <!-- Main content section. Main is a useful landmark element. -->
         <main>
-            {{.EntryContent}}
+        {{if .IncludeToC}}
+        {{with .Subsources}}
```

Comment:
Can't you just {{range .Subsources}} ? Also how does it work without a merge and only a digest ? There is no subsource IIRC

Reply:
The `{{with .Subsources}}` also checks that .Subsources is not nil and therefore prevents having an empty `<ul></ul>` in the html file if there is not subsources.

You're right about the digest without merge. The simplest of fixes would be to add
```go
source.Subsources = append(source.Subsources, source)
```
at the end of the `filterDigestGeneric()` function. it's a bit of a stretch but it is simpler to maintain that complicating the html template even more. What do you think? Are you okay with this solution I propose?

Reply:
> The {{with .Subsources}} also checks that .Subsources is not nil 
Perfect, I did not catch that.
> filterDigestGeneric
Yes I think it's better to patch this function

Reply:
Pushed the change 👍🏽